ui/desktop/src/components/ProviderGuard.tsx

```diff
@@ -73,6 +75,9 @@ export default function ProviderGuard({ children }: ProviderGuardProps) {
           setOpenRouterSetupState(null);
           setShowFirstTimeSetup(false);
           setHasProvider(true);
+          
+          // Navigate to chat after successful setup
```

Comment:
I think will need to check this running packaged, I had a hell of a time with things like this vs `just run-ui`